### ISSUE TITLE
Remove the term "Ajax" from the spec

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/aria#1160]
From the definition of Live region:

> These regions are not always updated as a result of a user interaction. This practice has become commonplace with the growing use of Ajax

This is such a dated term, would be good to take it out.

**Comment by carmacleod** [comment]
I'll remove the sentence containing Ajax, and another instance of "Ajax" in the project README file.

There are also a bunch of instances of "Ajax" in this test file:
https://github.com/w3c/aria/blob/master/testfiles/1.0/_functional/tree/js/prototype.js

Anybody know if this code is still running during the tests? And if so, is it a problem somehow, or should we just leave it?

**Comment by joanmarie** [comment]
This is merged. Anything else needed to address this issue?

**Comment by carmacleod** [comment]
Nope, nothing else needed. The question about test code was answered by Scott in https://github.com/w3c/aria/pull/1165#pullrequestreview-347523064.
Closing.

**Comment by carmacleod** [comment]
Argh! The Ajax sentence is still in the [definition of live region](https://w3c.github.io/aria/#dfn-live-region). Obviously the file I removed it from is being overwritten by some other file during the build.

Reopening this issue. I'll see if I can figure out where the "real" definition is coming from.

If anyone knows why there's duplicate content, please let me know... we should remove any duplication because it's a maintenance problem.

**Comment by carmacleod** [comment]
Ok, so the content isn't duplicated (thank goodness!).
It seems to have been just an accidental commit as part of https://github.com/w3c/aria/commit/ada7c70cee0e86bcb96b6f851a3e1b0d75772a2e
(@michael-n-cooper, I assume you did not intend to put Ajax back in to the live region definition?)

So I'm going to provide another PR to take the Ajax sentence back out.

**Comment by carmacleod** [comment]
Ok, ran around in circles and ended up committing the fix to the [aria-common](https://github.com/w3c/aria-common) repo. 😄 

Explanation begins here: https://github.com/w3c/aria/pull/1227#pullrequestreview-384132016

**TLDR;** changes in the aria/common directory need to get committed to the [aria-common](https://github.com/w3c/aria-common) repository.

Closing
